packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.ts

```diff
@@ -15,83 +15,145 @@ const spinButtonExtraClassNames = {
   buttonActive: 'fui-SpinButton__button_active',
 };
 
-const useRootStyles = makeStyles({
-  base: {
-    display: 'inline-grid',
-    gridTemplateColumns: `1fr 24px`,
-    gridTemplateRows: '1fr 1fr',
-    columnGap: tokens.spacingHorizontalXS,
-    rowGap: 0,
-    paddingRight: 0,
-    position: 'relative',
-    // Remove the border styles from react-input
-    ...shorthands.border('0'),
-    isolation: 'isolate',
-
-    // Apply border styles on the ::before pseudo element.
-    // We cannot use ::after since react-input uses that
-    // for the selector styles.
-    // Using the pseudo element allows us to place the border
-    // above content in the component which ensures the buttons
-    // line up visually with the border as expected. Without this
-    // there is a bit of a gap which can become very noticeable
-    // at high zoom or when OS zoom levels are not divisible by 2
-    // (e.g., 150% on Windows in Firefox)
-    // This is most noticeable on the "outline" appearance which is
-    // also the default so it feels worth the extra ceremony to get right.
-    '::before': {
-      content: '""',
-      boxSizing: 'border-box',
-      position: 'absolute',
-      top: 0,
-      right: 0,
-      bottom: 0,
-      left: 0,
-      ...shorthands.borderRadius(tokens.borderRadiusMedium),
-      pointerEvents: 'none',
-      zIndex: 10,
+// TODO(sharing) should these be shared somewhere?
+const fieldHeights = {
+  small: '24px',
+  medium: '32px',
+};
```

Comment:
FWIW there are a ton of styles that are similar/identical between components that aren't shared, which is fine. I don't think we need this TODO.